clang-tools-extra/docs/clang-tidy/checks/bugprone/unused-local-non-trivial-variable.rst

```diff
@@ -0,0 +1,45 @@
+.. title:: clang-tidy - bugprone-unused-local-non-trivial-variable
+
+bugprone-unused-local-non-trivial-variable
+==========================================
+
+Warns when a local non trivial variable is unused within a function.
+The following types of variables are excluded from this check:
+
+* trivial and trivially copyable
+* references and pointers
+* exception variables in catch clauses
+* static or thread local
+* structured bindings
+
+This check can be configured to warn on all non-trivial variables by setting `ExcludeTypes` to `.*`.
```

Comment:
Please follow 80-characters limit. Same in other places in this file.

Reply:
Fixed thanks.